uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/ScoreList.kt

```diff
@@ -61,6 +61,31 @@ class ScoreList {
         return result
     }
 
+    /**
+     * Returns the number of skips after the offset in the interval used to calculate
+     * the percentage of completed days.
+     *
+     * If skips are found in the interval, it is expanded until the interval has the size of the
+     * sum of the denominator and the number of skips within the interval.
+     */
+    @Synchronized
+    fun getNumberOfSkipsByInterval(
+        values: IntArray,
+        firstIndexToCheck: Int,
+        lastIndexToCheck: Int
```

Comment:
This parameter name is misleading, because in practice it's not necessarily the last index checked. Can you try to make it clearer for the reader what this does in which cases?